client/my-sites/stats/stats-followers/index.jsx

```diff
@@ -177,7 +177,7 @@ class StatModuleFollowers extends Component {
 							( emailData && emailData.subscribers.length !== emailData.total_email ) ) && (
 							<div key="view-all" className="module-expand">
 								<a href={ summaryPageLink }>
-									{ translate( 'View All', { context: 'Stats: Button label to expand a panel' } ) }
+									{ translate( 'View all', { context: 'Stats: Button label to expand a panel' } ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **85** times:
`translate( 'View all', { context: 'drafts'} )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/card-headings-form-labels-sentence-case)